src/test/java/com/micetr0/controller/MeasureControllerTest.java

```diff
@@ -0,0 +1,28 @@
+package com.micetr0.controller;
+
+import com.micetr0.model.Measure;
+import com.micetr0.model.Note;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertTrue;
+
+public class MeasureControllerTest {
+    private MeasureController controller;
+    private Measure model;
+
+    @Before
+    public void setUp()
+    {
+        controller = new MeasureController();
+        model = new Measure();
+        controller.setUp(model);
+        controller.updateMaxLen(1);
+    }
+
```

Comment:
need test cases

Reply:
Measure controller hasn't been implemented yet, will be taken care of in future pull request